sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -221,6 +225,7 @@ export class ServiceBusAdministrationClient extends ServiceClient {
    *
    */
   async getNamespaceProperties(
+    // eslint-disable-next-line @azure/azure-sdk/ts-naming-options
     operationOptions?: OperationOptions
```

Comment:
What's wrong here?

Reply:
We have this rule that the name of the type for the options in a public facing method should have the method name as suffix.